app/airq/models/clients.py

```diff
@@ -197,3 +207,45 @@ def maybe_notify(self) -> bool:
         db.session.commit()
 
         return True
+
+    #
+    # Stats getters
+    #
+
+    @classmethod
+    def get_total_num_sends(cls) -> int:
+        return (
+            cls.filter_phones().with_entities(func.sum(cls.num_alerts_sent)).scalar()
+            or 0
+        )
+
+    @classmethod
+    def get_total_num_subscriptions(cls) -> int:
+        return cls.filter_phones().filter(cls.alerts_disabled_at == 0).count()
+
+    @classmethod
+    def get_inactive_counts(cls):
+        windows = [1, 2, 3, 7, 30]
+        curr_time = cls.curr_ts()
+        groups = [
+            func.sum(
+                case(
+                    [
+                        (
+                            cls.last_activity_at > curr_time - (window * 24 * 60 * 60),
+                            literal_column("1"),
+                        )
+                    ],
+                    else_=literal_column("0"),
+                )
+            ).label("{} day{}".format(window, "s" if window > 1 else ""))
+            for window in windows
+        ]
```

Comment:
What is this "case, when" stuff, and "literal_column" ?

Reply:
It constructs SQL like the following:

```
SELECT 
   SUM(CASE WHEN condition_1 THEN 1 ELSE 0 END) as label_1,
   SUM(CASE WHEN condition_2 THEN 1 ELSE 0 END) as label_2,
   ... etc ..
FROM ..
```
So it lets us do a bunch of different counts in one query, basically.